### PR TITLE
feat: user profile

### PR DESCRIPTION
**Pull request by weronika28**
Please approve if everything is ok. I have created a user profile and the ability to edit it.

### CODE REVIEW

File: backend/src/main/java/pl/pollub/backend/auth/user/UsersRepository.java [hunk 1]
```diff
@@ -1,11 +1,18 @@
 package pl.pollub.backend.auth.user;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.web.bind.annotation.RestController;
 
 import java.util.Optional;
 
+@RestController
```
Comment (atakowiec) [inline backend/src/main/java/pl/pollub/backend/auth/user/UsersRepository.java:8]: probably to delete

File: frontend/src/main/java/pl/pollub/frontend/controller/LoginController.java [hunk 2]
```diff
@@ -126,6 +126,6 @@ private void handleValidLogin(JsonObject response) {
 
         authService.setUser(user);
 
-        screenService.switchTo("home");
+        screenService.switchTo("user");
```
Comment (atakowiec) [inline frontend/src/main/java/pl/pollub/frontend/controller/LoginController.java:129]: change to 'home', starting screen after login should be home

**Comment by weronika28** [comment]
everything corrected

File: frontend/src/main/java/pl/pollub/frontend/controller/MainViewController.java [hunk 3]
```diff
@@ -9,6 +9,7 @@
 import javafx.scene.layout.StackPane;
 import org.kordamp.bootstrapfx.BootstrapFX;
 import pl.pollub.frontend.FinanceApplication;
+import pl.pollub.frontend.annotation.View;
```
Comment (atakowiec) [inline frontend/src/main/java/pl/pollub/frontend/controller/MainViewController.java:12]: u forgot to optimize imports

**Review by atakowiec — CHANGES_REQUESTED** [review]

**Review by atakowiec — APPROVED** [review]
very cool

**Comment by atakowiec** [comment]
image

**Pull request merged**